src/backend/web/debug.log

```diff
@@ -0,0 +1 @@
+[1114/201901.723:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
```

Comment:
@zg3d:  why this is in here?

Reply:
Im not sure that on my end. I believe that is on the tests end